.travis.yml

```diff
@@ -19,51 +19,39 @@ services:
   - docker
 
 env:
+  # a build matrix for a version
   global:
     - UT3_OWNER=ut3
     - UT3_OWNER_PASSWORD=ut3
-    - UT3_OWNER_TABLESPACE=users
     - UT3_USER=ut3_user
     - UT3_USER_PASSWORD=ut3
-    - UT3_USER_TABLESPACE=users
+    - UT3_TABLESPACE=users
     # Target Branch and Directory for Deployment of Docs
     - PAGES_TARGET_BRANCH="gh-pages"
     - PAGES_VERSION_BASE="version3"
     - CACHE_DIR=$HOME/.cache
     # Database Env
     - SQLCLI="$HOME/sqlcl/bin/sql"
     - ORACLE_PWD="oracle"
-    - CONTAINER_IP=""
-    - CONNECTION_STR=""
-    - ORACLE_12cR1SE="12c-se-r1"
-    - ORACLE_11gR2XE="11g-xe-r2"
   matrix:
-    - ORACLE_VERSION=$ORACLE_12cR1SE
-    - ORACLE_VERSION=$ORACLE_11gR2XE
+    - ORACLE_VERSION="12c-se-r1" CONNECTION_STR='127.0.0.1:1521/ORCLPDB1'
+    - ORACLE_VERSION="11g-xe-r2" CONNECTION_STR='127.0.0.1:1521/XE'        DOCKER_OPTIONS='--shm-size=1g'
+
 
 cache:
+  pip: true
   directories:
     - $CACHE_DIR
+    - node_modules
```

Comment:
You don't need to cache the node_modules dir, since we only use it to download sqlcl. The downloaded file is cached, so next time we won't need to install those dependencies again.

Caching it will only increment the cache fetch time.